satpy/etc/composites/modis.yaml

```diff
@@ -12,15 +21,15 @@ composites:
       modifiers: [sunz_corrected]
     standard_name: true_color
 
-  true_color:
+  true_color_crefl:
```

Comment:
Please reset this to be `true_color` and to use `rayleigh_corrected` instead of `rayleigh_corrected_crefl`. Feel free to leave the `rayleigh_corrected_crefl` definition above. Make sure that this true color composite still loads successfully.